gateway/events.go

```diff
@@ -76,10 +77,10 @@ type RequestGuildMembersCommand struct {
 	GuildIDs []discord.GuildID `json:"guild_id"`
 	UserIDs  []discord.UserID  `json:"user_ids,omitempty"`
 
-	Query     string `json:"query,omitempty"`
-	Limit     uint   `json:"limit,omitempty"`
-	Presences bool   `json:"presences"`
-	Nonce     string `json:"nonce,omitempty"`
+	Query     option.NullableString `json:"query"`
+	Limit     option.NullableUint   `json:"limit"`
```

Comment:
Limit should also have `,omitempty` just because the official client sometimes excludes it. Can we check if omitting this field will still make them work for bots?

Reply:
If not, an idea would be to not omit it unless `Limit == -1`, then we can explicitly document them for non-bot users. This way, we prioritize bot support.